src/mam4xx/mo_sethet.hpp

```diff
@@ -481,6 +481,10 @@ void sethet_detail(
     int mm2 = wetdep_map[mm];
     Kokkos::parallel_for(Kokkos::TeamVectorRange(team, ktop),
                          [&](int kk) { het_rates(kk, mm2) = 0.0; });
+  }
+  Kokkos::fence();
+  for (int mm = 0; mm < gas_wetdep_cnt; mm++) {
+    int mm2 = wetdep_map[mm];
```

Comment:
@overfelt @odiazib @mjs271 : I was getting an error (`sethet: het_rates (wet dep) not set for het reaction number`)   [see line 498 of this file] and I added this fence. Please let me know if there is a better way to fix it. I believe the error is due to a race condition where `abort` becomes > 0.

Reply:
We should use `team.team_barrier()`, as this part of the code is within a parallel_for.

Reply:
Since you are modifying the code anyway, doing a ThreadVectorRange inside a TeamThreadRange as on line 298 is the way to go. And, yes, team_barrier() is the right call in this case. 

A team_barrier() right before the abort check on line 496 might be overkill, but there are already five team_barrier()s in this function, maybe one more would not hurt.